fdbserver/MockPeekCursor.h

```diff
@@ -21,39 +21,102 @@
 #ifndef FDBSERVER_MOCK_PEEK_CURSOR
 #define FDBSERVER_MOCK_PEEK_CURSOR
 
+#include <ostream>
 #include "fdbserver/LogSystem.h"
 
+struct VersionedMessage {
```

Comment:
I'm a little nervous about this name because `BackupWorker.actor.cpp` has a struct by the same name. That struct should really be in an unnamed namespace to avoid name collisions, but if the two structs ever get linked into the same binary, things could go very badly.

Reply:
I agree. Not sure if we want to add namespace for this one as well.